pages/ksphere/kommander/1.1.0-beta/tutorials/duplicating-cluster/index.md

```diff
@@ -10,22 +10,25 @@ In this tutorial you are going to learn how to create a second production cluste
 
 ## Previous Setup
 
-We have already set up a project for the production clusters of this team, let's take a look at it.
+We have already set up a project for this team's production clusters in a Development workspace, let's take a look at it.
 
 ![Creating an E-Commerce project](/ksphere/kommander/1.1.0-beta/img/tutorial-ecommerce-project.png)
 
-This project configuration will match every production cluster of our E-Commerce team; Let's give them admin access to the cluster.
-For this we need an admin role, which we will define as the permission to do everything on the cluster:
+This project configuration will ensure all of the E-Commerce team's clusters have matching configuration. Let's give the team admin access to the workspace.
+We have defined the developer access control group beforehand from the [Identity Providers page](https://docs.d2iq.com/ksphere/kommander/1.1.0-beta/operations/identity-providers/#groups) at the global level. We will create a policy to bind the developers group to the default Workspace Admin Role, which will give all developers access.
```

Comment:
The access control instructions previously were ambiguous (and out of date with our new RBAC set up). Is admin workspace permissions what we want here? There's no default project admin role (that isn't a kommander role) that I can see on the daily cluster which makes me wonder whether 'project admin' persona is something we support/suggest?